.gitignore

```diff
@@ -0,0 +1,101 @@
+# Logs
```

Comment:
should `.vscode` be in here?

Reply:
I debated this and decided it would be just so the function setup is there for local running.  But I'm happy to remove it if you feel strongly it's wrong